src/components/notifications/static/static.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import Static from './static';
+
+describe('Static', () => {
+  it('should pass data-attributes all way to the DOM', () => {
+    const { container } = render(
+      <Static data-foo="bar" type="info">
+        A message
+      </Static>
+    );
+
+    expect(container.querySelector("[data-foo='bar']")).toBeInTheDocument();
+  });
+
+  it.each(['error', 'info', 'success', 'warning'])(
+    'should render notification of type `%s`',
+    type => {
+      const { container } = render(<Static type={type}>A message</Static>);
+
+      expect(container).toMatchSnapshot();
```

Comment:
Don't like this kind of tests too much, but don't know what else can be tested for this kind of dumb presentational component.

Reply:
Yeah, me neither. I would not test that at all (as stated in https://github.com/commercetools/ui-kit/pull/232#discussion_r233527322).

What we're missing are Visual Regression Tests. This is what we really would like to test here. I'm trying to introduce it to ui-kit before I leave but it requires some coordination because it comes with a price-tag (as we need to use a paid service for it). I've just completely left out any tests which attempt to test the visuals (like classnames) recently. I want Visual Regression Testing and I don't want to deal with writing tests that attempt to catch these visual regressions (using classnames) which actually won't catch it in most cases (e.g. because somebody edits the css itself). 

TLDR: I'd just remove the tests if their only purpose is to test the styling.

Reply:
But, I would think about switching to CSS-in-JS approach and using snapshots again. That can be a nice substitute to the full-fledged VRT screenshot-based approach, as combination of DOM-nodes and styles kind of garanties specific appearance. Buy paying the price of more tight coupling to the implementation, we will get some obvious advantages: better performance, and getting rid off all fonts related issues.

Reply:
> and getting rid off all fonts related issues. 

We will experiment with Percy soon who have the font issues and other stuff figured out. They run super fast too because they run in parallel. I would not add snapshot tests in the meantime.